doc/book/src/development/release.md

```diff
@@ -0,0 +1,29 @@
+# Making a release
+
+As a part of [PR #199][199], we now have infrastructure in place to make releases automatically.
+This automation can be triggered by running a workflow dispatch on [the `Build container images`
+workflow][workflow].  This workflow takes the version to be tagged and released as its only input.
+
+This workflow will produce a draft release with autogenerated release notes.  It is recommended that
+important features get highlighted in the release notes for a release.  `Kustomize` manifests will
+be uploaded as release artifacts, which can be later consumed by [infra-deployments].
+
+# Deploying a release to `infra-deployments`
```

Comment:
I'd say that's good for now. In future we can add more details for the scenario in which changes to `base` manifests are part of the release. This will -to me- require another temporary folder (like `base-releaseversion`) to avoid production to be impacted. The promotion of production will be as easy as rename the temporary folder `base-releaseversion` to `base`).

This would be way clearer after we do the first releases, so I'd document that procedure as part of future dedicated stories.

For now I'd say we can just add a note here to not forget.

I mean something like:
> Note: ....

WDYT?

Reply:
Yep, this kind of thing was why I was talking about it in the first place.  I'll add in a note to flesh out details once this actually matters for a release.